lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,30 @@
+const Node = require('../node');
+
+
+// Returns true if the nodes are constant terms with the same constant and has an exponent each
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*' || node.args[1].op === '/') {
+    return false;
+  }
+  const args = node.args;
+  if (args.some(n=> Node.PolynomialTerm.isPolynomialTerm(n))) {
+    return false;
+  }
+  if (!args.every(n => Node.ConstantTerms.isConstantTerm(n))) {
```

Comment:
I think a helper function that takes something of the form c^n or c can return the base and exponent

e.g. 10^3 => returns 10 and 3
e.g. 10 => returns 10 and 1

with polynomial terms there's also a coefficient which makes everything much more messy which is why there's a whole class for it